autogluon/inference/docker/1.1/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,76 @@
+ARG PYTHON_VERSION=3.10.8
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference:2.1.0-cpu-py310-ubuntu20.04-sagemaker
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get autoremove -y \
+ && apt-get install tesseract-ocr -y \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+ARG AUTOGLUON_VERSION=1.1.0
+
+# Upgrading pip and installing/updating Python dependencies
+# Comments are added to explain the reason behind each update
+RUN pip install --no-cache-dir -U --trusted-host pypi.org --trusted-host files.pythonhosted.org pip \
+ && pip install --no-cache-dir -U setuptools wheel \
+ && pip uninstall -y dataclasses \
+ && pip install --no-cache-dir -U numpy numba \
+# Install AutoGluon, ensuring no vulnerable dependencies are left behind
+ && pip install --no-cache-dir -U autogluon==${AUTOGLUON_VERSION} \
+# Update requests to a secure version that fixes CVE-2023-32681
+ && pip install --no-cache-dir -U requests>=2.31.0 \  
+ && mim install -q mmcv==2.1.0 --timeout 600 \
+ && pip install --no-cache-dir -U mmdet>=3.0.0 \
+ && pip install --no-cache-dir -U "PyMuPDF" \
```

Comment:
Let's remove this one since it is no longer needed

Reply:
done